### PR TITLE
Fix for APIMF-1613

### PR DESCRIPTION
**Pull request by rddheath**
https://www.mulesoft.org/jira/browse/APIMF-1613

### CODE REVIEW

File: modules/ROOT/pages/design-named-examples.adoc [hunk 1]
```diff
@@ -224,3 +243,6 @@ familyName: "Mukai"
 givenName: "Kyung-won"
 familyName: "Park"
 ----
+
+API Designer simply ignores the header and uses the examples. However, if you planned to validate your specification with other RAML processors, these two fragments would generate errors or warnings.
```
Comment (jstoiko) [inline modules/ROOT/pages/design-named-examples.adoc]: > with other RAML processors

I would add “any” before “other”

> would generate errors or warnings

I would replace “would” by “should” here. I would replace “or warnings” by “as per RAML 1.0 specification”.

Comment (rddheath) [reply modules/ROOT/pages/design-named-examples.adoc]: @jstoiko , I've changed the paragraph to this:
"API Designer simply ignores the header and uses the examples. However, if you validate a specification with any other RAML processors, NamedExample fragments that define only one example should generate errors, in accord with the RAML 1.0 specification."

File: modules/ROOT/pages/design-named-examples.adoc [hunk 2]
```diff
@@ -195,7 +195,7 @@ fullName:
 
 If you do not want to use more than one example and you do not want to name your single example, use the `example` facet instead of the `examples` facet.
 
-Here, the `example` facet includes `fragment1.raml`. Neither the facet nor the fragment names the example.
+Here, the `example` facet includes `fragment1.yaml`. Neither the facet nor the fragment names the example.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-named-examples.adoc]: To what does "Here," refer? Should we say "In this example," or "In this case," instead?

File: modules/ROOT/pages/design-named-examples.adoc [hunk 3]
```diff
@@ -195,7 +195,7 @@ fullName:
...
 
 If you wanted the `example` facet to point to `fragment2.raml`, you would need to change the filename in the facet. You could not include both fragments at the same time.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-named-examples.adoc]: If you want...you must change.... You cannot....
[present tense]

File: modules/ROOT/pages/design-named-examples.adoc [hunk 4]
```diff
@@ -208,9 +208,28 @@ types:
...
+.fragment2.yaml
+----
+givenName: "Kyung-won"
+familyName: "Park"
+----
+
+
+[NOTE]
+====
+According to the RAML 1.0 specification, a NamedExample fragment that defines only one example is not valid. Therefore, you should not use NamedExample fragments when you are using single, unnamed examples.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-named-examples.adoc]: Is this ref. to RAML still valid, given the file name extension change? Just double-checking....

File: modules/ROOT/pages/design-named-examples.adoc [hunk 5]
```diff
@@ -208,9 +208,28 @@ types:
...
+
+For example, the following NamedExample fragments would be invalid:
```
Comment (kbostrom) [inline modules/ROOT/pages/design-named-examples.adoc]: would be invalid > are invalid

GLOBALLY standardize to simple present tense as much as possible without impairing meaning; if present works just as well, try to use it. It's so much easier to translate.

File: modules/ROOT/pages/design-named-examples.adoc [hunk 6]
```diff
@@ -224,3 +243,6 @@ familyName: "Mukai"
 givenName: "Kyung-won"
 familyName: "Park"
 ----
+
+API Designer simply ignores the header and uses the examples. However, if you validate a specification with any other RAML processors, NamedExample fragments that define only one example should generate errors, in accord with the RAML 1.0 specification.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-named-examples.adoc]: Omit "simply"

Comment (kbostrom) [reply modules/ROOT/pages/design-named-examples.adoc]: should generate > generate, if technically accurate

File: modules/ROOT/pages/design-named-examples.adoc (hunk 4 above)
Comment (rddheath) [reply modules/ROOT/pages/design-named-examples.adoc]: Yes. I've revised the beginning of the note to make more clear that the note explains why the example uses .yaml files and not .raml files.

**Pull request merged**